web/src/main/kotlin/no/nav/su/se/bakover/web/routes/sak/SakJson.kt

```diff
@@ -55,16 +56,37 @@ internal data class SakJson(
                 )
             },
             utbetalingerKanStansesEllerGjenopptas = utbetalingstidslinje().tidslinje.let {
```

Comment:
Hadde gjerne sett at det var `Sak` som utledet svare på dette spørsmålet og ikke `Sak.toJson()`. Føles også mer fornuftig da jeg tror vi trenger mer info enn bare utbetalingene for å finne det faktiske svaret på spørsmålet (kombinasjon av tilstand på vedtak og utbetaling).

Reply:
Ja, enig, det var litt snodig at det lå i json-mapperen, jeg kan se om jeg rekker å få flytta det over til Sak i løpet av dagen.